docker-compose.yml

```diff
@@ -1,25 +1,25 @@
 services:
-   palworld:
-      image: thijsvanloef/palworld-server-docker:latest
-      restart: unless-stopped
-      container_name: palworld-server
-      stop_grace_period: 30s # Set to however long you are willing to wait for the container to gracefully stop
-      ports:
-        - 8211:8211/udp
-        - 27015:27015/udp # Required if you want your server to show up in the community servers tab 
-      environment:
-         - PUID=1000
-         - PGID=1000
-         - PORT=8211 # Optional but recommended
-         - PLAYERS=16 # Optional but recommended
-         - SERVER_PASSWORD=worldofpals # Optional but recommended
-         - MULTITHREADING=true
-         - RCON_ENABLED=true
-         - RCON_PORT=25575
-         - TZ=UTC
-         - ADMIN_PASSWORD=adminPasswordHere
-         - COMMUNITY=false  # Enable this if you want your server to show up in the community servers tab, USE WITH SERVER_PASSWORD!
-         - SERVER_NAME=World of Pals
-         - SERVER_DESCRIPTION=palworld-server-docker by Thijs van Loef
-      volumes:
-         - ./palworld:/palworld/
+  palworld:
+    image: thijsvanloef/palworld-server-docker:latest
+    restart: unless-stopped
+    container_name: palworld-server
+    stop_grace_period: 30s # Set to however long you are willing to wait for the container to gracefully stop
+    ports:
+      - 8211:8211/udp
+      - 27015:27015/udp # Required if you want your server to show up in the community servers tab
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - PORT=8211 # Optional but recommended
+      - PLAYERS=16 # Optional but recommended
+      - SERVER_PASSWORD=worldofpals # Optional but recommended
+      - MULTITHREADING=true
+      - RCON_ENABLED=true
+      - RCON_PORT=25575
+      - TZ=UTC
+      - ADMIN_PASSWORD=adminPasswordHere
+      - COMMUNITY=false  # Enable this if you want your server to show up in the community servers tab, USE WITH SERVER_PASSWORD!
+      - SERVER_NAME=World of Pals
+      - SERVER_DESCRIPTION=palworld-server-docker by Thijs van Loef
+    volumes:
+      - ./palworld:/palworld/
```

Comment:
We currently use 3 spaces in this YAML only. 
I do support this update to 2 spaces since all the other YAMLs are 2.